lua/spec/neotest-go/init_spec.lua

```diff
@@ -33,6 +33,17 @@ assert:register(
   "assertion.has_property.negative"
 )
 
+require("neotest").setup({
```

Comment:
Do we need same for `utils_spec.lua`?

Reply:
Tests were failing for me because I have different args set in my own config.

This seemed to fix the test expectations but also didn't overwrite them after I had run tests.

Reply:
Does it makes sense to put it to master if it's specific for you? 🤔

Reply:
It's not specific to me. It's specific to the test expectations. With no `args` specified in my own config, multiple tests fail. including
```
Fail	||	build_spec build specification for many_table_test.go
            ...re/nvim/site/pack/packer/start/neotest/lua/nio/tests.lua:27: Test task failed with message:
            The coroutine failed with this message:
            ...ensla/repos/neotest-go/lua/spec/neotest-go/init_spec.lua:358: Expected objects to be the same.
            Passed in:
            (string) 'cd /Users/jstensla/repos/neotest-go/neotest_go && go test -v -json  ./...'
            Expected:
            (string) 'cd /Users/jstensla/repos/neotest-go/neotest_go && go test -v -json  -count=1 -timeout=60s ./...'
            stack traceback:
            	[C]: in function 'error'
            	...e/pack/packer/start/plenary.nvim/lua/luassert/assert.lua:51: in function 'same'
            	...ensla/repos/neotest-go/lua/spec/neotest-go/init_spec.lua:358: in function <...ensla/repos/neotest-go/lua/spec/neotest-go/init_spec.lua:349>

            stack traceback:
            	...re/nvim/site/pack/packer/start/neotest/lua/nio/tests.lua:27: in function <...re/nvim/site/pack/packer/start/neotest/lua/nio/tests.lua:12>
```

What I meant was that they also fail if I specify whatever I want for my configuration (I setup a coverage arg), but that's (somewhat) beside the point. I am guessing you have these `args` set for yourself if the tests pass without this setup in the test

Reply:
I have a doubts because we have already defined plugin on [L6](https://github.com/nvim-neotest/neotest-go/pull/65/files#diff-da925398f9c19e2ebc126c22e92cd3f013ce501e4e787bd788466f431c768000R6)